actionlist/action_list_items.js

```diff
@@ -66,7 +66,7 @@ export default class ActionListItems extends Component {
     super(props);
     const ds = new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 !== r2 });
     this.state = {
-      dataSource: ds.cloneWithRows(JSON.parse(lib.getListJSON())),
+      dataSource: ds.cloneWithRows(JSON.parse(lib.getProcessInstanceJSON())),
```

Comment:
This function doesn't exist anymore (also, I know you didn't write it, but line 12 can be simplified to use ES6 imports to get you just what you need: `const lib = require('./commons.js');` -> `import { processInstanceJSON } from './common';`